news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,239 @@
+
+*written by [\_DUSK\_](https://osu.ppy.sh/users/6092181)*
+
+## <a id="catch"></a> osu!catch
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/chocolate.jpg)](https://osu.ppy.sh/beatmapsets/1061800)
+
+Spring is coming, wanna play a map with spring flavour? 'hot chocolate' by [JBHyperion](https://osu.ppy.sh/users/4879508) should be one of the best choices here. Needless to say, the high-quality mapping style is always a mark for featured artist maps, this map is no exception. 
+
+Despite the extraordinary clean and fresh flows, the mapper also tried some new features here - extending sliders and 1/8 Hyperdashes. These innovative patterns bring the map to the next level and make the map stands out as well. Please make a cup of hot chocolate and enjoy this fantastic map!
+
+*written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/loving.jpg)](https://osu.ppy.sh/beatmapsets/1080025)
+
+'How on earth did eroge get into spotlights?' may be a question that pops up when first encountering this set. Despite the general stigma against the genre, [Luminate](https://osu.ppy.sh/users/4778689)'s "Loving Trip" does a great job of following the song. 
+
+Slower parts of the song keep things interesting with varied rhythm and patterns, while higher density sections have liberal use of antiflow that contrasts well. This is one of the few sets that I've genuinely had fun playing this year, and I hope you give it a shot as well.
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/lights.jpg)](https://osu.ppy.sh/beatmapsets/1044462)
+
+[Kukkai](https://osu.ppy.sh/users/7811952)'s "Lights of Muse" has a certain feeling behind it that is fairly uncommon these days, reminiscent of something that would have easily had a play count in the hundred thousand if it was ranked a few years ago. While the use of patterns and rhythm is fairly simple, the whole set is at a consistent level of fun that is hard to come by.
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/sweet.jpg)](https://osu.ppy.sh/beatmapsets/738169)
+
+[Sorcerer](https://osu.ppy.sh/users/1913190) and [ZiRoX](https://osu.ppy.sh/users/200768) has something sweet to offer in this season's Spotlights!
+
+Starting from the bottom we have Cup, Salad and Platter all made by Sorcerer, and they lead the player through the learning stages of the streams and how to control the jumps. If you can already get full combo on the Salad then the next step is to learn how to handle the Hyperjumps and time your normal jumps at the end of the streams in Platter. Very well designed difficulty curve.
```

Comment:
you can probably condense this into one sentence about difficulty curve

Reply:
```suggestion
Starting from the bottom we have Cup, Salad and Platter all made by Sorcerer, and they lead the player through the learning stages of the streams and jumps. Platter introduces Hyperjumps and even more advanced stream patterns.
```